frontend/model/state.js

```diff
@@ -101,6 +169,7 @@ sbp('sbp/selectors/register', {
       if (!ourIdentityContractId || !state[ourIdentityContractId]?.groups) return
       Object.entries(state[ourIdentityContractId].groups).map(([groupID, { hasLeft }]: [string, Object]) => {
         if (hasLeft || !state[groupID]?.chatRooms) return undefined
+        if (Array.isArray(contractIDHints) && !contractIDHints.includes(groupID)) return undefined
```

Comment:
Isn't the point of the `wrappedUpdateFn` to handle this?

Reply:
I'll add a comment. It's not sufficient because `wrappedUpdateFn` is filtering contracts by type, while this is filtering out contracts (groups) that haven't been recently added.